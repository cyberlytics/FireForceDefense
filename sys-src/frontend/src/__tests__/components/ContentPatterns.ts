import { mount } from '@vue/test-utils';
import ContentPatterns from '@components/contentPatterns.vue';

describe('ContentPatterns.vue', () => {
    test('content patterns match snapshot', () => {
        const wrapper = mount(ContentPatterns);
        expect(wrapper.element).toMatchSnapshot();
    });
});
