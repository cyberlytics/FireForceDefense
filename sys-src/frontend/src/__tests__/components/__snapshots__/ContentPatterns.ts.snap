// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentPatterns.vue content patterns match snapshot 1`] = `
<defs>
  <pattern
    height="1"
    id="content-Loeschkran"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/content-Loeschkran.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="content-Loeschschiff"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/content-Loeschschiff.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="content-Loeschzeppelin"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/content-Loeschzeppelin.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="content-Loeschtrupp"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/content-Loeschtrupp.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="content-Loeschturm"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/content-Loeschturm.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="content-Basis"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/content-Basis.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="content-Haus"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/content-Haus.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="content-Baumgruppe"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/content-Baumgruppe.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="content-Steinbrocken"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/content-Steinbrocken.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="content-Brandreste"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/content-Brandreste.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="content-Totholz"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/content-Totholz.svg"
      y="0"
    />
  </pattern>
</defs>
`;
